circle.yml

```diff
@@ -230,6 +230,27 @@ commands:
           working_directory: ~/build
           command: bin/fizzy-bench-internal --benchmark_repetitions=<<parameters.repetitions>> --benchmark_min_time=<<parameters.min_time>> --benchmark_out=internal-<<parameters.out_dir>>
 
+  fetch_spectests:
+    description: "Download and preprocess the WebAssembly spec tests"
+    steps:
+      - install_wabt
+      - run:
+          name: "Download spectest files"
+          working_directory: "~"
+          command: |
+            git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-tests-backported-20210212 --depth 1 --single-branch
```

Comment:
The previous version extracted to `wasm_spec` specifically, but this also uses the same directory, I guess that is the internal directory name?

Reply:
It will extract to `~/wasm-spec` because this is the repo name (as previously, just not explicit this time).